### PR TITLE
Repository maintenance

### PR DESCRIPTION
**Pull request by KochTobi**
The PR contains changes to unify the repository configuration and it's README across QBiC

### CODE REVIEW

**Comment by KochTobi** [comment]
As this tool is not supposed to be used outside of QBiC and the help messages are elaborate, I would argue that the section `How To Use` is not necessary in the README file

**Comment by jenniferboedker** [comment]
> As this tool is not supposed to be used outside of QBiC and the help messages are elaborate, I would argue that the section `How To Use` is not necessary for the README file

I would argue it would help people inside QBiC that are not familiar with the tool (like me) to understand it. Can you please add it?

**Comment by KochTobi** [comment]
> > As this tool is not supposed to be used outside of QBiC and the help messages are elaborate, I would argue that the section `How To Use` is not necessary for the README file
> 
> I would argue it would help people inside QBiC that are not familiar with the tool (like me) to understand it. Can you please add it?

I do not know what to write there as the information is already contained in `How To Run` IMHO

File: README.md [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
-| `USER_DB_HOST`                      | The URL to the host of the user database containing the database name  |
-| `USER_DB_USER_NAME`                 | The database user name                                                 |
-| `USER_DB_USER_PW`                   | The database user password                                             |
-
-## Run the app
+## How to run
+
+To run the application, first make sure you provide a correct [configuration](#configuration).
 
 Checkout the latest code from `main` and run the Maven goal `spring-boot:run`:
```
Comment (jenniferboedker) [inline README.md]:
```suggestion
Checkout the latest code from `main` and run the maven goal `spring-boot:run`:
```

**Comment by jenniferboedker** [comment]
don't forget the license

File: README.md [hunk 2]
```diff
@@ -1,35 +1,59 @@
...
+[![license](https://img.shields.io/github/license/qbicsoftware/sample-status-reporter)](https://github.com/qbicsoftware/sample-status-reporter/blob/main/LICENSE)
+![language](https://img.shields.io/badge/language-groovy,%20java-blue.svg)
+![framework](https://img.shields.io/badge/framework-spring-blue.svg)
 
-```
+</div>
+
+## System Integration
+
+[//]: # (TODO what role does this application play in the infrastructure?)
```
Comment (jenniferboedker) [inline README.md]: here is still a todo

Comment (KochTobi) [reply README.md]: :+1: addressed but forgot to remove

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**